crates/exex/src/manager.rs

```diff
@@ -81,13 +81,20 @@ impl ExExHandle {
         )
     }
 
-    /// Reserves a slot in the `PollSender` channel and sends the notification if the slot was
+    /// Attempts to prepare the sender to receive a value.
+    /// See also [PollSender::poll_reserve]
+    fn poll_reserve(&mut self, cx: &mut Context<'_>) -> Poll<Result<(), PollSendError<CanonStateNotification>>> {
+        self.sender.poll_reserve(cx)
+    }
+
+    /// Sends a slot in the `PollSender` channel and sends the notification if the slot was
     /// successfully reserved.
```

Comment:
please correct this comment - it just sends the notification, it does not do anything w slots or reservations